sql/item_mods.sql

```diff
@@ -1728,13 +2235,86 @@ INSERT INTO `item_mods` VALUES (10764,370,1);
+-- INSERT INTO `item_mods` VALUES (10783,LEGION:MACC,4);
+-- INSERT INTO `item_mods` VALUES (10783,LEGION:OCC QUICKEN SPELLCASTING%,2);
+INSERT INTO `item_mods` VALUES (10784,25,5); -- DHANURVEDA RING, Accuracy+5 Ranged Attack+5 "Subtle Blow"+8
+INSERT INTO `item_mods` VALUES (10784,24,5); -- RACC
+INSERT INTO `item_mods` VALUES (10784,289,8); -- SUBTLE
+INSERT INTO `item_mods` VALUES (10785,1,15); -- PROVOCARE RING, DEF:15 VIT+3 Enmity+5
+INSERT INTO `item_mods` VALUES (10785,10,3); -- VIT
+INSERT INTO `item_mods` VALUES (10785,27,5); -- ENMITY
+INSERT INTO `item_mods` VALUES (10786,5,25); -- MEDIATORS RING, MP+25 Magic Accuracy+4 Enmity-5
+INSERT INTO `item_mods` VALUES (10786,30,4); -- MACC
+INSERT INTO `item_mods` VALUES (10786,27,-5); -- ENMITY
 INSERT INTO `item_mods` VALUES (10787,375,3);
+INSERT INTO `item_mods` VALUES (10788,30,5); -- MAQUETTE RING, Magic Accuracy+5 Legion:"Magic Atk. Bonus"+5 Adds "Refresh" effect
+-- INSERT INTO `item_mods` VALUES (10788,LEGION:MAB,5);
+-- INSERT INTO `item_mods` VALUES (10788,LEGION:ADD REFRESH EFFECT,1);
+INSERT INTO `item_mods` VALUES (10789,25,-4); -- FOLHARDI RING, Accuracy-4 Attack+9
+INSERT INTO `item_mods` VALUES (10789,23,9); -- ATT
+INSERT INTO `item_mods` VALUES (10790,1,4); -- EPHEDRA RING, DEF:4 Healing magic skill +7 Enhances "Cursna" effect
+INSERT INTO `item_mods` VALUES (10790,112,7); -- HEALING SKILL
+INSERT INTO `item_mods` VALUES (10790,310,1); -- EN CURSNA
+INSERT INTO `item_mods` VALUES (10791,1,5); -- HAOMAS RING, DEF:5 Healing magic skill +8 Enhances "Cursna" effect
+INSERT INTO `item_mods` VALUES (10791,112,8); -- HEALING SKILL
+INSERT INTO `item_mods` VALUES (10791,310,1); -- EN CURSNA
+INSERT INTO `item_mods` VALUES (10792,1,4); -- SAIDA RING, DEF:4 Enhances effect of "Cursna" received
+-- INSERT INTO `item_mods` VALUES (10792,ENHANCE EFFECT OF CURSNA RECEIVED,1);
+INSERT INTO `item_mods` VALUES (10793,1,5); -- ESHMUNS RING, DEF:5 Enhances effect of "Cursna" received
+-- INSERT INTO `item_mods` VALUES (10793,ENHANCE EFFECT OF CURSNA RECEIVED,1);
+-- INSERT INTO `item_mods` VALUES (10794,ENHANCE HOLY WATER EFFECT,1); -- BLENMOTS RING, Enhances "Holy Water" effect
+-- INSERT INTO `item_mods` VALUES (10795,ENHANCE HOLY WATER EFFECT,1); -- BLENMOTS RING +1, Enhances "Holy Water" effect
+INSERT INTO `item_mods` VALUES (10797,23,10); -- DAGAZ RING, Attack+10 Ranged Attack+10 Enmity-1
+INSERT INTO `item_mods` VALUES (10797,24,10); -- RATT
+INSERT INTO `item_mods` VALUES (10797,27,-1); -- ENMITY
 INSERT INTO `item_mods` VALUES (10798,2,70);
 INSERT INTO `item_mods` VALUES (10798,27,5);
 INSERT INTO `item_mods` VALUES (10798,255,5);
+INSERT INTO `item_mods` VALUES (10799,8,3); -- SLJOR RING, STR+3 "Double Attack"+1% "Slow"+2%
+INSERT INTO `item_mods` VALUES (10799,288,1); -- DATT
+-- INSERT INTO `item_mods` VALUES (10799,SLOW%,2);
```

Comment:
Use the gear haste mod (384) with a negative value for this. Remember it's /1024 so if it's 2% slow you put in 20.